### DISCUSSION TITLE
Create Bullet List from Selected

### DISCUSSION THREAD

**Discussion by spacec0de** [outline/outline#1967] [Ideas]
Would it be possible to enable creation of an Ordered List/Unordered List from a highlighted selection?

This would make it easier to format text copied and pasted into Outline from other sources. Unless I'm missing something, it appears that creation of an OL/UL is only possible line by line?

**Comment by tommoor** [comment]
Yes! You can actually do this today with a keyboard shortcut:

- `Shift+Ctrl+7` = convert to todo list
- `Shift+Ctrl+8` = convert to unordered list
- `Shift+Ctrl+9` = convert to ordered list

It looks like this is not clearly documented, so thanks for the prompt!
